app/services/requirements/video_embed_checker.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Requirements
+  class VideoEmbedChecker
+    YOUTUBE_HOST = "youtube.com"
+    YOUTU_HOST = "youtu.be"
+
+    def pre_embed_issues(title, url)
```

Comment:
Just a minor thing, but I was a bit confused reading the tests and wondered if these would be better as keyword arguments?

Reply:
Sure - I was on the fence at the time. How does it look now?